DashAI/front/src/components/experiments/ConfigureModelsStep.jsx

```diff
@@ -45,7 +45,12 @@ function ConfigureModelsStep({ newExp, setNewExp, setNextEnabled }) {
 
   const getModelSchema = async () => {
     try {
-      const schema = await getModelSchemaRequest(selectedModel);
+      let schema;
+      if (selectedModel === "DistilBertTransformer") {
+        schema = await getModelSchemaRequest("DistilBERT");
+      } else {
+        schema = await getModelSchemaRequest(selectedModel);
+      }
```

Comment:
This looks like a future bug. 
It is not possible to no hardcode `DistilBERT`?
If not, add a card indicating that we should solve this problem.

Reply:
It will be fixed in another pr.